### ISSUE TITLE
False code in utils._get_extension causing TypeError

### ISSUE THREAD

**Issue by Pandaaaa906** [LonamiWebs/Telethon#1159]
There is fault in utils.py causing TypeError.
Here is the traceback.

```python
File "H:\Python36\lib\site-packages\telethon\client\uploads.py", line 253, in send_file
    supports_streaming=supports_streaming
  File "H:\Python36\lib\site-packages\telethon\client\uploads.py", line 483, in _file_to_media
    as_image = utils.is_image(file) and not force_document
  File "H:\Python36\lib\site-packages\telethon\utils.py", line 665, in is_image
    match = re.match(r'\.(png|jpe?g)', _get_extension(file), re.IGNORECASE)
  File "H:\Python36\lib\site-packages\telethon\utils.py", line 653, in _get_extension
    return ('.' + kind) if kind else ''
TypeError: must be str, not bool

```
Here is the location of the false code
```python
def _get_extension(file):
    # omitted code...
    elif isinstance(file, io.IOBase) and file.seekable():
        kind = imghdr.what(file) is not None  # <---------this line causing the problem
        return ('.' + kind) if kind else ''
    # omitted code...
```

**Comment by Lonami** [comment]
Would you like to submit a pull request?

**Comment by Lonami** [comment]
This was fixed by #1119 but is not yet released in PyPi.
